v2/bindings/main.go

```diff
@@ -337,7 +337,6 @@ import (
 	"github.com/solarwinds/snap-plugin-lib/v2/plugin"
 	"github.com/solarwinds/snap-plugin-lib/v2/runner"
 )
-import "C"
```

Comment:
import is defined twice. hence removed redundant.

Reply:
Could you explain this? Where is the second definition?

Reply:
Please check Line 325